README.md

```diff
@@ -1 +1,145 @@
+    - [Key Features](#key-features)
+  - [:movie_camera: Project Presentation](#project-presentation)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Global-Summit] <a name="about-project"></a>
+
+
+**[Global-Summit]** Is a website can be used for confranse or glopbal events and runs smoothly on different devices and screen sizes
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+Built useing Bootstrap and font awesome.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Responsive design]**
+- **[Hover effects]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Project Presntation -->
+
+## :movie_camera: Project Presentation <a name="project-presentation"></a>
+
+Walking through portfolio outline.
+
+- [Project Presentation Link] [https://www.loom.com/share/5a39affb783a4b66974ec21aa711bd76](https://www.loom.com/share/5a39affb783a4b66974ec21aa711bd76)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link] [[https://jlberbesi.github.io/Microverse-first-module.github.io/](https://jlberbesi.github.io/Capstone_project_m1.github.io/)]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+Use Git or checkout with SVN using the web URL.
+https://github.com/
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Jose Berbesi**
+
+- GitHub: [@jlberbesi](https://github.com/jlberbesi)
+- Twitter: [@imberbesi](https://twitter.com/imberbesi)
+- LinkedIn: [in/jlberbesi](https://www.linkedin.com/in/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- [ ] **[Animation and Hover effects]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Templete used in project provided by (Microverse).
+- Original design idea by Cindy Shin in Behance.
```

Comment:
- [ ] It would be a good thing to also include the link of the original design. This is so that visitors who like your design and would like to create it on their own, so that they can also get the original design guidelines and design it themselves.

![image](https://user-images.githubusercontent.com/86318284/229046899-af46a4e5-58f4-4a5c-9474-e7330826f8e3.png)

Reply:
Hi @Nemwel-Boniface . It was a good catch. I think I already added successfully.

Reply:
The attribute to Cindy Shin was well included :heavy_check_mark: